### PR TITLE
Adding hover to the toggle view buttons

### PR DESCRIPTION
**Pull request by srsharon**
image

### CODE REVIEW

File: modules/web/scss/modules/ballerina-editor.scss [hunk 1]
```diff
@@ -191,10 +191,39 @@
...
+            content: '';
+            display: block;
+            height: 0;
+            width: 0;
+            position: absolute;
+            top: -14px;
+            font-size:12px;
+        }
+        div[title]:hover:before {
+            visibility: visible;
```
Comment (dakshika) [inline modules/web/scss/modules/ballerina-editor.scss:223]: Is there any specific reason for adding this?

File: modules/web/scss/modules/ballerina-editor.scss [hunk 2]
```diff
@@ -191,10 +191,39 @@
...
+            content: attr(title);
+            position: absolute;
+            top: -3em;
+            left: -1em;
+            visibility: hidden;
+            color: #fff;
+            text-align: center;
+            background-color: #303030;
+            width: 100px;
+
```
Comment (dakshika) [inline modules/web/scss/modules/ballerina-editor.scss:208]: remove unwanted whitespace

File: modules/web/src/plugins/ballerina/views/view-button.jsx [hunk 3]
```diff
@@ -30,7 +30,7 @@ class ViewButton extends React.Component {
                 className={`view-split-view-btn btn-icon ${active}`}
                 onClick={this.props.onClick}
             >
-                <div className='bottom-label-icon-wrapper'>
+                <div className='bottom-label-icon-wrapper' title={this.props.title}>
```
Comment (dakshika) [inline modules/web/src/plugins/ballerina/views/view-button.jsx:33]: Why can't we use same prop label in the view button?

**Comment by srsharon** [comment]
Changes have been done on PR https://github.com/ballerina-lang/ballerina/pull/4583
Closing this PR

**Pull request closed without merging**